### PR TITLE
Adding common-variables.yml required for the codeql pipeline

### PR DESCRIPTION
**Pull request by vijayrkn**
- [ ] You've read the [Contributor Guide](../CONTRIBUTING.md) and [Code of Conduct](../CODE-OF-CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

**For PRs which target a specific extension within UA, changes won't be approved by a member of the `dotnet-upgrade-assistant-admin` group until a member of the code owners of the extension has approved, when required.**

## Description
Detail 1
Detail 2

Addresses #bugnumber (in this specific format)

### CODE REVIEW

File: eng/common-variables.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      value: True
+    - name: _SignType
+      value: real
+    # DotNet-Blob-Feed provides: dotnetfeed-storage-access-key-1
+    # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
+    # DotNet-HelixApi-Access provides: HelixApiAccessToken
+    - group: DotNet-Blob-Feed
+    - group: Publish-Build-Assets
+    - group: DotNet-HelixApi-Access
+    - group: SDL_Settings
```
Comment (MattGal) [inline eng/common-variables.yml:27]: This is too much of a direct copy of Arcade's values, you don't need all these variable groups and SDL_Settings is specific to Arcade-y repos.   
For CodeQL purposes you just need variables for these values:
``` 
_TsaBugAreaPath 
_TsaCodebaseAdmin
_TsaInstanceURL
_TsaIterationPath
_TsaNotificationEmail
_TsaProjectName
_TsaTemplateName
```

Comment (vijayrkn) [reply eng/common-variables.yml]: Updated the public variables.

**Review by MattGal — APPROVED** [review]
Looks fine, some nitpicks:
- At this point you could probably just directly include these in the other yaml file
- Not sure if _TsaCodebaseAdmin / _TsaNotificationEmail / _TsaTemplateName are actually required here but you'll definitely find out running the pipeline...

**Comment by vijayrkn** [comment]
> Looks fine, some nitpicks:
> 
> * At this point you could probably just directly include these in the other yaml file
> * Not sure if _TsaCodebaseAdmin / _TsaNotificationEmail / _TsaTemplateName are actually required here but you'll definitely find out running the pipeline...

Agreed. Let me go ahead, merge and see if there are failures & I can fix them all up in a followup. Thanks for taking a look.

**Comment by vijayrkn** [comment]
@mjrousos - Can I get an ack on the PR?

**Pull request merged**